apps/service_acquire/lib/acquire/query/where/temporal.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Acquire.Query.Where.Temporal do
+  import Acquire.Query.Where.Functions
+
+  @spec to_queryable(
+          dataset_id :: String.t(),
+          subset_id :: String.t(),
+          after_time :: String.t(),
+          before_time :: String.t()
+        ) :: Acquire.Queryable.t()
+  def to_queryable(_, _, "", ""), do: nil
+
+  def to_queryable(dataset_id, subset_id, after_time, before_time) do
+    with {:ok, dictionary} <- Acquire.Dictionaries.get(dataset_id, subset_id, "dictionary") do
+      Dictionary.get_by_type(dictionary, Dictionary.Type.Timestamp)
+      |> Enum.map(&Enum.join(&1, "."))
+      |> Enum.map(&to_queryable(&1, after_time, before_time))
+      |> or_clause()
+    end
```

Comment:
Let's talk about the `Acquire.Dictionaries.get` pattern that we did with WKT. We should change it here or there so it's the same.